### ISSUE TITLE
opentelemetry-instrumentation-rails gem is broken

### ISSUE THREAD

**Issue by johnnyshields** [open-telemetry/opentelemetry-ruby#489]
This line in your `Gemfile` will cause your app not to boot (version 0.9.0). Please fix and cut a new release.

```
gem 'opentelemetry-instrumentation-rails', require: true
```

**Comment by robertlaurin** [comment]
Hey @johnnyshields, just making sure this is what you're running into 👇 

```
/ruby/2.6.5/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require': cannot load such file -- opentelemetry-instrumentation-rails (LoadError)
```

**Comment by johnnyshields** [comment]
Yep that's the one!

**Comment by johnnyshields** [comment]
I've confirmed it's fixed on 0.10.0. Closing issue.
